lib/rspec/parameterized/core.rb

```diff
@@ -39,6 +39,10 @@ def initialize(arg_names, &block)
         #     end
         #
         def where(*args, &b)
+          if @parameter
+            format = ->(b) { b.source_location&.join(':') || '?' }
```

Comment:
`where` can be called without a block so the source location would be missing.

Reply:
On the other hand, redefining `where` is OKish and used at GitLab like this:

```ruby
context "foo" do
  where(...)
  with_them do
    it 'allow' do
    end
  end

  where(...)
  with_them do
    it 'disallow' do
    end
  end
```

Removing this warning again.